### PR TITLE
create rails api to send random greetings 

### PR DESCRIPTION
**Pull request by saied2035**
### Project requirements

 - I created a new Rails API app called 'hello-rails-back-end'.
 - I initialized the project with Git.
 - The project has a Postgres database set up.  
 - I created a table for storing messages and I created 5 different greetings. 
 - I created an API endpoint that selects a random greeting from your table.

Live Demo:
[link](https://sleepy-brook-35791.herokuapp.com/messages)

This is the link to the React app PR:
[link](https://github.com/saied2035/hello-react-front-end/pull/1)

Thanks for reviewing my PR. 😄

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @saied2035, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have:- 
1. In the README file for the frontend, provide a link to the Rails API (backend app).
1. In the README file for the backend, you should find a link to the frontend part.

Make sure to add both.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: db/migrate/20220712084832_create_messages.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  def change
+    create_table :messages do |t|
+      t.string :data
+      t.timestamps
+    end
+    Message.create(data:'Greeting 1')
+    Message.create(data:'Greeting 2')
+    Message.create(data:'Greeting 3')
+    Message.create(data:'Greeting 4')
+    Message.create(data:'Greeting 5')
```
Comment (rustam-tolipov) [inline db/migrate/20220712084832_create_messages.rb]: - [x] [OPTIONAL] You can put this hard-coded data into seeds.rb file and run `rake db:seed` to generate them, to make your migration folder much cleaner.

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse: [microverse community](https://github.com/microverseinc)
+
+## 📝 License
+
+This project is [MIT](./MIT) licensed.
```
Comment (rustam-tolipov) [inline README.md:68]: - [x] [OPTIONAL] I think you just forgot to put MIT license on your project. Kindly include it as well.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Kamba56👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by saied2035** [comment]
**@Rustamxon7**  Thanks for reviewing my PR. I appreciate your optional suggestions I did them. Thanks again! 😄

**Pull request merged**